src/components/com_tjdashboard/administrator/tables/widgets.php

```diff
@@ -38,6 +38,13 @@ public function __construct(&$db)
 	 */
 	public function check()
 	{
+		if ((!empty($this->params)) && (json_decode($this->params) === null))
```

Comment:
Create a new validator instead of checking like this 
https://docs.joomla.org/Server-side_form_validation

Reply:
If we remove json parameter field after 5-6 month then , is this really need to Create a new validator

Reply:
Thats a valid point. This is fine for now.